charts/policies/user-group-psp/values.yaml

```diff
@@ -0,0 +1,29 @@
+---
+kind: ClusterAdmissionPolicy
+metadata:
+  name: user-group-psp
+  namespace: default # used when kind is AdmissionPolicy
+spec:
+  module:
+    repository: "registry://ghcr.io/kubewarden/policies/user-group-psp"
+    tag: "v0.2.0"
+  mode: "protect"
+  mutating: true
+  rules:
+    -
```

Comment:
Added rules into values instead of hardcoding them, so they can be changed in the yaml tab of the policy.